soc/Kconfig

```diff
@@ -11,3 +11,19 @@ osource "$(SOC_DIR)/$(ARCH)/Kconfig"
 osource "$(SOC_DIR)/$(ARCH)/*/Kconfig"
 
 endmenu
+
+# The helper symbols below are put here due to an unusual setup: The simulated
```

Comment:
Need a better spot for this, we should not have soc specific kconfigs in a general file such as  soc/Kconfig. 
Also, I would do "select SOC_COMPATIBLE_NRF" in the SOC_FAMIL_NRF config instead of this.

Reply:
> Also, I would do "select SOC_COMPATIBLE_NRF" in the SOC_FAMIL_NRF config instead of this.

Ok.

> Need a better spot for this, we should not have soc specific kconfigs in a general file such as soc/Kconfig.

Do you have a better place in mind? (please consider this comment from Ulf: https://github.com/zephyrproject-rtos/zephyr/pull/9988#discussion_r222670030 )

Reply:
ok, can't think of a better solution now, will revisit.